src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMAT_DATE = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate firstDay = LocalDate.parse(dateFrom, FORMAT_DATE);
+        LocalDate lastDay = LocalDate.parse(dateTo, FORMAT_DATE);
+        StringBuilder totalResult = new StringBuilder();
+        totalResult.append("Report for period ")
+                .append(dateFrom).append(" - ")
+                .append(dateTo).append("\n");
+        for (int i = 0, namesLength = names.length; i < namesLength; i++) {
```

Comment:
let's use for-each loop for better readability

Reply:
done